client/my-sites/plans/main.jsx

```diff
@@ -157,10 +162,17 @@ class Plans extends Component {
 		);
 	}
 
+	renderEcommerceTrialPage() {
+		return <div className="plans__ecommerce-trial-wrapper">{ this.renderPlansMain() }</div>;
+	}
+
 	render() {
 		const { selectedSite, translate, canAccessPlans, currentPlan, domainAndPlanPackage } =
 			this.props;
 
+		const currentPlanSlug = selectedSite.plan.product_slug;
```

Comment:
@nelsonec87 This line seems to be causing an error when you try to create a new site via the sidebar `Add New Site` while you're in the /plans section.

<img width="420" alt="Screenshot 2023-02-03 at 18 28 29" src="https://user-images.githubusercontent.com/2749938/216655953-8d8cc741-48f8-456a-a8d7-1c023757bf7e.png">

Reply:
I can confirm that this is broken in Production

Reply:
I have raised the issue here
https://github.com/Automattic/wp-calypso/issues/72990

Reply:
Working on a fix in #72992.